app/lib/person_capital_subtotals.rb

```diff
@@ -1,29 +1,36 @@
 class PersonCapitalSubtotals
-  def initialize(total_vehicle: 0.0, total_liquid: 0,
-                 total_mortgage_allowance: 0.0, total_non_liquid: 0.0, total_property: 0.0,
-                 disputed_property_disregard: 0.0, pensioner_capital_disregard: 0.0,
-                 main_home: PropertySubtotals.new,
-                 additional_properties: [], disputed_non_property_disregard: 0.0)
+  # This (and other similar classes) has 2 use cases: (a) fully-populated and (b) blank
+  # This structure helps enforce that so that e.g. tests are updated when the structure changes
+  class << self
+    def blank
+      new(total_vehicle: 0.0, total_liquid: 0,
+          total_mortgage_allowance: 0.0, total_non_liquid: 0.0,
+          disputed_property_disregard: 0.0, pensioner_capital_disregard: 0.0,
+          properties: [], disputed_non_property_disregard: 0.0, disputed_capital: 0.0, non_disputed_capital: 0.0)
+    end
+  end
+
+  def initialize(total_vehicle:, total_liquid:,
+                 total_mortgage_allowance:, total_non_liquid:,
+                 disputed_property_disregard:, pensioner_capital_disregard:,
+                 properties:, disputed_non_property_disregard:, disputed_capital:, non_disputed_capital:)
     @total_vehicle = total_vehicle
     @total_liquid = total_liquid
     @total_mortgage_allowance = total_mortgage_allowance
     @total_non_liquid = total_non_liquid
-    @total_property = total_property
-    @disputed_property_disregard = disputed_property_disregard
     @pensioner_capital_disregard = pensioner_capital_disregard
-    @main_home = main_home
-    @additional_properties = additional_properties
+    @disputed_property_disregard = disputed_property_disregard
     @disputed_non_property_disregard = disputed_non_property_disregard
+    @properties = properties
+    @disputed_capital = disputed_capital
```

Comment:
Should this be `disputed_non_property_capital`? The `total_non_disputed_capital` method makes it looks like property is double-counted otherwise.